resources/definitions/hellbot_magna_2_500.def.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 2,
+    "name": "Hellbot Magna 2 500",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+        "platform": "Hellbot_Magna_2_500.obj",
+        "platform_texture": "Magna2_500.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_magna_2_500_extruder_0"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Hellbot Magna 2 500" },
+        "machine_width": {
+            "default_value": 500
+        },
+        "machine_height": {
+            "default_value": 500
+        },
+        "machine_depth": {
+            "default_value": 500
+        },        
+        "machine_heated_bed": {
+            "default_value": true
+        },
+        "machine_center_is_zero": {
+            "default_value": false
+        },
+        "machine_extruder_count": {
+            "default_value": 1
```

Comment:
As far as I could tell, this printer has two extruders? Looking at the [product page](https://hellbot.xyz/magna-2/) of the Magna 2, it always seems to have two. Or is there a single-extruder edition out there?

Reply:
Hmm, I don't seen any option for that in your store either: https://store.hellbot.xyz/18-magna-2

Of course, any dual extruder can also print with a single extruder. Cura already provides an option to reduce the number of extruders in a printer. I don't see why you'd have to provide that option duplicate using a different set of profiles. It'd prevent the user from using the settings they used for one print for another print that uses multiple extruders.